components/src/dynamo/vllm/tests/test_vllm_logging.py

```
@@ -125,6 +127,49 @@ def test_dyn_log_does_not_affect_vllm_level(monkeypatch):
+        ("info", logging.INFO),
+        ("warn", logging.WARNING),
+        ("info,dynamo_trtllm=debug", logging.DEBUG),
+        ("warn,dynamo_trtllm=trace", logging.DEBUG),
+    ],
+)
+def test_python_log_level_mapping(filters, expected):
+    assert python_log_level_mapping(filters) == expected
+
+
+def test_toml_logging_config_keeps_python_debug_available(monkeypatch):
+    monkeypatch.setenv("DYN_LOGGING_CONFIG_PATH", "/tmp/dynamo-logging.toml")
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/dynamo-logging.toml"

(S108)